src/Twig/Extension/SonataDoctrineORMAdminExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class SonataDoctrineORMAdminExtension extends AbstractExtension
```

Comment:
This should probably be defined in another package… the admin bundle, maybe?

Reply:
@greg0ire no problem, I will make a PR and move it to main bundle then, but I guess then this PR will be failing until we get new release of Admin Bundle

Reply:
One question: why not the core-bundle?

Reply:
Yeah I'm pretty undecided... maybe we should create a twig bundle like @core23 suggested. This might be handy in other bundle that do not require the admin bundle as well.

Reply:
Well, that is up to you guys, I don't mind moving this somewhere else.

Reply:
I think moving it to the core would be indeed better, we should not block you with a split-the-core decision.

Reply:
ok, will close https://github.com/sonata-project/SonataAdminBundle/pull/4833 and move it to CoreBundle later.